packages/@aws-cdk/aws-stepfunctions-tasks/lib/invoke-function.ts

```diff
@@ -22,6 +22,8 @@ export interface InvokeFunctionProps {
  * The Lambda function Arn is defined as Resource in the state machine definition.
  *
  * OUTPUT: the output of this task is the return value of the Lambda Function.
+ *
+ * @deprecated Use `RunLambdaTask`
```

Comment:
Can we also mark `InvokeFunctionProps` as deprecated in a similar way

Reply:
done